timesketch/frontend/src/views/SketchExplore.vue

```diff
@@ -459,6 +459,11 @@ export default {
       // Update with selected fields
       this.currentQueryFilter.fields = this.selectedFields
 
+      // Escape defanged indicators (such as timesketch(.)org or 8.8.8[.]8)
+      this.currentQueryString = this.currentQueryString.replace(/\((?:\.|dot)\)/g, '.')
+      this.currentQueryString = this.currentQueryString.replace(/\[(?:\.|dot)\]/g, '.')
+      this.currentQueryString = this.currentQueryString.replace(/hxxp(s?):\/\//g, 'http$1://')
```

Comment:
what if that's what you are searching for? I mean what if a string contains hxxp://foobar.com, but now you silently replace it without really telling me, isn't that confusing for the end user?